src/Common/src/System/Net/Http/Http2/CopyToAspNetCore.cmd

```diff
@@ -0,0 +1,14 @@
+@ECHO OFF
```

Comment:
why not instead setting up a code mirror?

Reply:
We will be eventually (follow up PR).

Reply:
These  scripts are for inner loop development, allowing us to test shared code in both repos before committing.

Reply:
We will soon consolidate the repositories and paths in these files will be wrong. If this is just for inner loop development I would instead of the relative path use git to determine the repo root.

Reply:
> use git to determine the repo root

How do you do that in this context?

These are easy enough to fix after the repo gets restructured.

Reply:
@ViktorHofer, we'd appreciate any guidance on #41787 on how to set up a codemirror.

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/tests/StressTests/HttpStress/load-corefx-testhost.ps1#L25-L27

Reply:
cc @Anipik for the code mirror question

Reply:
Mostly it was to avoid the cost of maintaining it as its not that stable.

Reply:
If you prefer a script, would Powershell be more platform agnostic?

Reply:
Elsewhere we've provided both cmd and sh scripts for different platforms and only called into a shared powershell script if the logic got complex. This case is pretty simple so anyone willing to contribute a matching sh script would be welcome.

Reply:
> anyone willing to contribute a matching sh script would be welcome.

I can send one tomorrow.

Reply:
-> https://github.com/dotnet/corefx/pull/42425 for the bash-version